tls/s2n_connection.c

```diff
@@ -1534,3 +1541,43 @@ S2N_RESULT s2n_connection_dynamic_free_in_buffer(struct s2n_connection *conn)
 
     return S2N_RESULT_OK;
 }
+
+bool s2n_connection_check_io_status(struct s2n_connection *conn, s2n_io_status side)
```

Comment:
nit - The s2n_io_status variable was renamed to `status` in the header, which I think makes more sense:
```suggestion
bool s2n_connection_check_io_status(struct s2n_connection *conn, s2n_io_status status)
```